Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -675,7 +727,8 @@ public class DependencyResolver<
     func resolveSubtree(
         _ container: Container,
         subjectTo allConstraints: ConstraintSet,
-        excluding allExclusions: [Identifier: Set<Version>]
+        excluding allExclusions: [Identifier: Set<Version>],
+        containers: Set<Identifier>
```

Comment:
I don't think this parameter should be necessary here -- this is already resolving a particular container, so it shouldn't actually matter what other containers the client requested (they will eventually get pulled in via the main loop).